plugins/CoreHome/Categories/VisitorsCategory.php

```diff
@@ -9,10 +9,26 @@
 namespace Piwik\Plugins\CoreHome\Categories;
 
 use Piwik\Category\Category;
+use Piwik\Piwik;
+use Piwik\Url;
 
 class VisitorsCategory extends Category
 {
     protected $id = 'General_Visitors';
     protected $order = 5;
     protected $icon = 'icon-reporting-visitors';
+
+    public function getHelp()
+    {
+        $visitsLogUrl = Url::getQueryStringFromParameters([
+            'category' => 'General_Visitors',
+            'subcategory' => 'Live_VisitorLog',
+        ]);
+        $visitsLogUrl = '<a href="?' . htmlspecialchars($visitsLogUrl) . '" rel="noreferrer noopener">';
```

Comment:
That link doesn't work. Guess it should be more like this:
```
<a href="#" onclick="this.href=broadcast.buildReportingUrl('category=General_Visitors&subcategory=Live_VisitorLog')">
```

Reply:
👍 thanks, modified